observability/otel/otel-configuring-metrics-for-monitoring-stack.adoc

```diff
@@ -15,7 +15,7 @@ As a cluster administrator, you can configure the OpenTelemetry Collector custom
 include::modules/otel-config-send-metrics-monitoring-stack.adoc[leveloffset=+1]
 include::modules/otel-config-receive-metrics-monitoring-stack.adoc[leveloffset=+1]
 
-[id="additional-resources_otel-configuring-metrics-for-monitoring-stack"]
+[id="additional-resources_otel-configuring-metrics-for-monitoring-stack_{context}"]
```

Comment:
One last instance where the manual context should be removed 😅

Reply:
😅 How did I miss this one?!